README.md

```diff
@@ -27,6 +27,83 @@ The Looker SDK has several parts:
+
+To simplify configuration tasks (and to help ensure you *never* commit your credentials into a source code repository) future SDKs provided by Looker will use an `.ini` format to save/retrieve their API configuration settings.
+
+### Using the yarn/node-based generator
+
+If you don't have `yarn` installed already, you'll need to [install](https://yarnpkg.com/en/docs/install) it.
+
+After yarn is installed, just run `yarn` from your terminal window/command line, and it will download or update all the packages required to run the code generator. You can look at [package.json](package.json) to see what resources are required to run the code generator.
+
+Run the generator with the command:
+
+```bash
+yarn ts-node generate.ts
+```
+
+The generator will:
+
+* read the Looker API configuration(s) from the `looker.ini` file.
+
+  * **Note**: There should be at most 2 entries in `looker.ini`: one for API version 3.1. and one for 3.0. Because 3.1 is a superset of 3.0, you really only need 3.1
+
+* download (if the file is not already present) the Looker API specification file(s) from the configured Looker server(s)
+
+* convert (if the converted file is not already present) the downloaded Swagger 2 specification file(s) to OpenAPI 3.x
+
+* lint check the OpenAPI 3.x file(s)
+
+* call the OpenAPI code generator for each active language configured in [`targetLanguages.ts`](targetLanguages.ts)
+
+  * Comment out any language you don't want to generate, or uncomment or add the languages you do want to generate.
+
+When the generator completes successfully, the output will be similar to:
+
+```plain-text
+api (created by the generator. This and its subfolders can be recreated with the script on demand.)
+  3.1 (api version in the configuration file)
+    csharp
+    kotlin
+    python
+    r
+```
+
+If you have the OpenAPI 3.x file available locally, you can use Speccy from the command line to search and explore the specification file. For example:
+
+```bash
+yarn speccy serve Looker.3.1.oas.json
+```
+
+will start a web server on `http://localhost:5000` that allows you do browse and search the local specification file for API 3.1.
+
+**Tip**: search for `query` in the UI and see what you get!
+
+**Note:** If you're unable to download the API specification file because you're using an instance of Looker that is not secured and errors are being thrown, you can explicitly turn off TLS verification in Node with a command like:
+
+```bash
+export NODE_TLS_REJECT_UNAUTHORIZED="0" && yarn ts-node generate.ts
+```
+
+### Using the bash script generators
+
 In this repository, [`prepare.sh`](prepare.sh) automates these steps for the OpenAPI generator. For people who still wish to use the deprecated swagger-based code generator, [`swagger.sh`](swagger.sh) is also provided.
```

Comment:
We should start pruning these deprecated things aggressively.  We can mention them in doc with links to the git commit history where they still exist.  Or set up tags to mark where they are.

Reply:
Agreed on pruning. I'd like to give this update a few weeks to cook/be consumed before deprecating the bash scripts.

Reply:
👍